src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -61,7 +61,10 @@ public SkillController(SkillRepository skillRepository) {
 
         Skill skill = skillOptional.get();
         skill.setName(skillDetails.getName());
```

Comment:
Can we move all these lines somewhere else?

Reply:
Yep, I can move to another method or even better I am thinking of using a mapper tool like http://mapstruct.org/.
What do you prefer?

Reply:
the mapper tool sounds good. There is also a cool library cool doozer that you might want to check.

Reply:
Ok,I will look over the mapper tools. I will refactor some method/classes. I hope It does not affect the other two PR. (mainly Erika's)

Reply:
I remember for the persons-api we just assigned the id from the optional to the object which we got in the request body and then we saved that instead of reassigning each value.
[See here](https://github.com/ioet/bpm-people-api/blob/master/src/main/java/com/ioet/bpm/people/boundaries/PersonController.java#L61) just the password field is an exception here...

Reply:
Good catch @oemel09. I just changed it.
It is Done